### PR TITLE
Feature/reconcile h1s

### PR DESCRIPTION
**Pull request by noahmanger**
Adjusts classes and markup throughout to accommodate new header styles from https://github.com/18F/fec-style/pull/520

### CODE REVIEW

**Comment by jenniferthibault** [comment]
@adborden surfacing the question @noahmanger left in https://github.com/18F/fec-style/issues/490#issuecomment-250344537 about which legal side templates you've started that this would impact. I can think of `legal-archived-mur.html` `legal-current-mur.html` and `legal-enforcement-landing.html` 

Are there other I'm missing?

File: openfecwebapp/templates/election-lookup.html [hunk 1]
```diff
@@ -60,8 +60,8 @@ <h2 class="t-ruled--bottom t-ruled--bold">Search</h2>
     </div>
     <div class="content__section--extra">
       <div class="usa-width-one-half">
-        <div class="section__heading">
-          <h2 class="heading__title u-no-margin">Results:</h2>
+        <div class="heading--section">
```
Comment (noahmanger) [inline openfecwebapp/templates/election-lookup.html]: Yes!

**Comment by noahmanger** [comment]
Ok, addressed those issues, and I think I got all the places in legal templates that are currently in there, including the MUR pages.

**Comment by jenniferthibault** [comment]
Quick check on next steps. Seem like these require a check by Aaron before any merging, is that right?

**Comment by noahmanger** [comment]
Yeah, that would be good.

**Pull request merged**